content/setup/overview.md

```diff
@@ -26,6 +26,16 @@ REMOTE_CONFIG=https://github.com?client_id=....&client_secret=....
 
 Please configure your [GitHub](../github) integration (or [GitLab](../gitlab), [Bitbucket](../bitbucket), [Gogs](../gogs)) before your proceed. Drone will not start until you have configured your version control integration.
 
+Build logs are only available to authenticated users with pull access to the
+source repository. In some deployments it may be desirable to make build
```

Comment:
this impacts not just the logs, but also the repository page with the build history. Perhaps something like this: 

> Public mode allows every repository, including private repositories, and their build results to be publicly viewable without authentication. This should only be enabled on private networks, where access to drone is restricted, to avoid leaking sensitive information.